batchflow/tests/model_save_load_test.py

```diff
@@ -39,26 +39,23 @@ def pipelines(model_setup_images_clf):
         make pipelines for model loading and saving, that are compatible with given `model_class`
         """
         def _pipelines(model_class):
-            config = {}
+            config = {'model_class': model_class}
             dataset, model_config = model_setup_images_clf(data_format='channels_first')
 
-            save_pipeline = (Pipeline()
-                             .init_variable('predictions', default=[])
-                             .init_model('dynamic', model_class, 'model', model_config)
-                             .to_array(dtype='float32')
-                             .predict_model('model', B.images,
-                                            fetches='predictions',
-                                            save_to=V('predictions', mode='a')))
-            load_pipeline = (Pipeline()
-                             .init_variable('predictions', default=[])
-                             .to_array(dtype=C('dtype'))
-                             .predict_model('model', B.images,
-                                            fetches='predictions',
-                                            save_to=V('predictions', mode='a')))
-
-            save_pipeline = (save_pipeline << dataset) << config
-            load_pipeline = (load_pipeline << dataset) << config
-            return save_pipeline, load_pipeline
+            train_pipeline = (Pipeline()
+                .init_model('model', model_class, 'dynamic', model_config)
+                .to_array(dtype='float32')
+                .train_model('model', inputs=B('images'), targets=B('labels'), outputs='loss')
+            )
+
+            predict_pipeline = (Pipeline()
+                .init_variable('predictions', default=[])
+                .to_array(dtype='float32')
+                .predict_model('model', inputs=B.images,
+                            outputs='predictions',
+                            save_to=V('predictions', mode='a'))
+            )
+            return (train_pipeline + predict_pipeline) << dataset << config, predict_pipeline << dataset << config
```

Comment:
plz use `<<` with pipline definitions above, now this line is very hard to read

Please also consider using consistent batch components definition, either `B.comp`, or `B('comp')`